lib/status_code.rb

```diff
@@ -8,23 +8,21 @@ module StatusCode
   I18n.backend.load_translations
 
   def self.decode(options)
-    if options[:code] && options[:receiver]
-      message = find_message(options)
-      message unless message == ''
-    end
+    find_message(options) if options[:code] && options[:receiver]
   end
 
-  private
+  private_class_method
 
   def self.find_message(options)
     receiver = options[:receiver].to_s.downcase
     code     = options[:code].to_s
     gateway  = options[:gateway].to_s.downcase
     locale   = options[:locale].to_s.downcase.to_sym if options[:locale]
```

Comment:
Can be simplified like this:
`locale   = options[:locale].to_s.downcase.to_sym || :en`